### PR TITLE
Create Sym Integration Tile

### PR DESCRIPTION
**Pull request by llam15**
### What does this PR do?
- Adds a new Integration Tile for Sym

### Motivation
- We are publishing a new Datadog app/integration for Sym

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests (N/A)
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes
The App UUID is `d81d1dd3-d5e8-4373-98a6-f95b797b3f9d`

### CODE REVIEW

File: sym/manifest.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Send Sym Audit Logs to Datadog",
+    "title": "Sym",
+    "media": [
+      {
+        "media_type": "video",
+        "caption": "A Sym Overview Video",
+        "image_url": "images/sym_video_thumbnail.png",
+        "vimeo_id": 123456789
```
Comment (llam15) [inline sym/manifest.json:18]: We have submitted our video to the Marketplace team, but are still waiting on a Vimeo ID.

File: sym/manifest.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        "support": "README.md#Support",
+        "changelog": "CHANGELOG.md",
+        "description": "Send Sym Audit Logs to Datadog",
+        "title": "Sym",
+        "media": [
+            {
+                "media_type": "video",
+                "caption": "A Sym Overview Video",
+                "image_url": "images/sym_video_thumbnail.jpg",
+                "vimeo_id": 123456789
```
Comment (eho1307) [inline sym/manifest.json:18]:
```suggestion
                "vimeo_id": 846654213
```

File: sym/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# CHANGELOG - Sym
+
+## 1.0.0 / 2023-07-17
+* [Added] Initial Release
```
Comment (eho1307) [inline sym/CHANGELOG.md:4]:
```suggestion
***Added***:

* Initial release
```

Updating the format to our new formatting.

File: sym/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# Sym Log Destination
+
+## Overview
+
+Sym is a platform for defining simple automations that turn just-in-time access policies into easy-to-operate workflows, executed in Slack. Define access flows in Terraform, customize and integrate with other systems in code, and then use our API or Slack App to request and approve/deny access. Simple!
+
+This integration enables customers to send Sym audit logs directly to Datadog via a Sym Log Destination. 
+These logs are sent in real time for every Event processed by the Sym platform, such as `request` or `approve`.
```
Comment (eho1307) [inline sym/README.md:8]:
```suggestion
Sym is a platform that allows you to define simple automations that turn just-in-time access policies into easy-to-operate workflows, executed in Slack. Define access flows in Terraform, customize and integrate with other systems in code, and then use our API or Slack App to request and approve/deny access.

This integration enables customers to send Sym audit logs directly to Datadog via a Sym Log Destination. 

These logs are sent in real time for every event processed by the Sym platform, such as `request` or `approve`.
```

File: sym/README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+## Setup
+
+### Installation
+
+To set up the Sym integration:
+1. Go to your [Datadog Integrations page](https://app.datadoghq.com/integrations) and click on the Sym tile.
+2. Click on "Install Integration".
+3. Datadog will redirect you to Sym to begin the OAuth authorization flow. Enter your Sym Org ID here to continue to log in to Sym.
+4. After authenticating, Sym will redirect you to Datadog to authorize Sym with the `api_keys_write` scope. After you authorize Sym, Sym will create an API Key in your Datadog organization, which will be used to send logs to Datadog directly.
+5. If Sym successfully connects with Datadog, a `sym_log_destination` Terraform resource will be displayed that should be copied into your Sym Terraform Configuration.
```
Comment (eho1307) [inline sym/README.md:19]:
```suggestion
1. From the Sym Datadog Integration tile, click on "Connect Accounts".
2. Datadog will redirect you to Sym to begin the OAuth authorization flow. Enter your Sym Org ID here to continue to log in to Sym.
3. After successfully authenticating, a `sym_log_destination` Terraform resource will display. Copy and paste this into your Sym Terraform Configuration.
```

Every single step isn't needed. I shortened the instructions to be clearer and more direct.

File: sym/README.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
+# Sym Log Destination
+
+## Overview
+
+Sym is a platform for defining simple automations that turn just-in-time access policies into easy-to-operate workflows, executed in Slack. Define access flows in Terraform, customize and integrate with other systems in code, and then use our API or Slack App to request and approve/deny access. Simple!
```
Comment (eho1307) [inline sym/README.md:5]:
```suggestion
[Sym][1] is a platform for defining simple automations that turn just-in-time access policies into easy-to-operate workflows, executed in Slack. Define access flows in Terraform, customize and integrate with other systems in code, and then use our API or Slack App to request and approve/deny access. Simple!
```

File: sym/README.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+### Validation
+
+After you have Terraformed your Datadog Log Destination, you can confirm its existence with:
+```
+ symflow resources list sym_log_destination
+```
+
+## Troubleshooting
+
+Need help? Contact us at support@symops.com.
```
Comment (eho1307) [inline sym/README.md:34]:
```suggestion
Need help? Contact us at [support@symops.com][2].

[1]: https://symops.com/
[2]: mailto:support@symops.com
[3]: https://docs.symops.com/docs/datadog

```

File: sym/README.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+### Configuration
+
+Check out the [Sym Docs](https://docs.symops.com/docs/datadog) for detailed instructions on how to configure your Datadog Log Destination in Terraform.
+
+### Validation
+
+After you have Terraformed your Datadog Log Destination, you can confirm its existence with:
+```
+ symflow resources list sym_log_destination
+```
```
Comment (eho1307) [inline sym/README.md:30]: Where does one check this?

File: sym/README.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+To set up the Sym integration:
+1. Go to your [Datadog Integrations page](https://app.datadoghq.com/integrations) and click on the Sym tile.
+2. Click on "Install Integration".
+3. Datadog will redirect you to Sym to begin the OAuth authorization flow. Enter your Sym Org ID here to continue to log in to Sym.
+4. After authenticating, Sym will redirect you to Datadog to authorize Sym with the `api_keys_write` scope. After you authorize Sym, Sym will create an API Key in your Datadog organization, which will be used to send logs to Datadog directly.
+5. If Sym successfully connects with Datadog, a `sym_log_destination` Terraform resource will be displayed that should be copied into your Sym Terraform Configuration.
+
+### Configuration
+
+Check out the [Sym Docs](https://docs.symops.com/docs/datadog) for detailed instructions on how to configure your Datadog Log Destination in Terraform.
```
Comment (eho1307) [inline sym/README.md:23]:
```suggestion
Check out the [Sym Docs][3] for detailed instructions on how to configure your Datadog Log Destination in Terraform.
```

File: sym/assets/dashboards/sym_overview.json [hunk 10]
```diff
@@ -0,0 +1,419 @@
+{
+    "title": "Sym",
+    "description": null,
```
Comment (eho1307) [inline sym/assets/dashboards/sym_overview.json:3]: Please add a short description describing what a customer can expect in this dashboard.

File: sym/assets/dashboards/sym_overview.json [hunk 11]
```diff
@@ -0,0 +1,419 @@
...
+                "title_align": "left",
+                "show_legend": true,
+                "legend_layout": "auto",
+                "legend_columns": [
+                    "avg",
+                    "min",
+                    "max",
+                    "value",
+                    "sum"
+                ],
```
Comment (eho1307) [inline sym/assets/dashboards/sym_overview.json:19]: Please add a logo to your dashboard.

File: sym/manifest.json [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+                "image_url": "images/define_deploy.jpg"
+            },
+            {
+                "media_type": "image",
+                "caption": "Request, approve, and deny anywhere in Slack",
+                "image_url": "images/request_approve.jpg"
+            }
+        ],
+        "classifier_tags": [
+            "Submitted Data Type::Logs"
```
Comment (eho1307) [inline sym/manifest.json:37]:
```suggestion
        "classifier_tags": [
            "Supported OS::Windows",
            "Supported OS::macOS",
            "Supported OS::Linux",
            "Category::Security",
            "Offering::Integration",
            "Submitted Data Type::Logs"
```
Added a few tags I thought were applicable.

File: sym/manifest.json [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+        "dashboards": {
+            "Sym Overview": "assets/dashboards/sym_overview.json"
+        },
+        "integration": {
+            "source_type_name": "sym",
+            "configuration": {},
+            "events": {
+                "creates_events": false
+            },
+            "service_checks": {
```
Comment (eho1307) [inline sym/manifest.json:50]:
```suggestion
            },
            "metrics": {
             "metadata_path": "metadata.csv",
             "check": [],
             "prefix": "sym."
           },
            "service_checks": {
```

Even if the integration is not sending in metrics, the metrics metadata.csv file with the pre-populated headers is still required.

Can you please add the metadata.csv file back? [https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#complete-the-necessary-integration-asset-files](https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#complete-the-necessary-integration-asset-files)

**Review by eho1307 — CHANGES_REQUESTED** [review]
Thanks for opening this PR! I've left comments inline. Let me know if you have any questions. You will also see other reviews roll in from other internal teams. Feel free to address them as they come in.

File: sym/manifest.json [hunk 14]
```diff
@@ -0,0 +1,64 @@
...
+                "media_type": "image",
+                "caption": "Define and deploy access rules in Terraform",
+                "image_url": "images/define_deploy.jpg"
+            },
+            {
+                "media_type": "image",
+                "caption": "Request, approve, and deny anywhere in Slack",
+                "image_url": "images/request_approve.jpg"
+            }
+        ],
```
Comment (eho1307) [inline sym/manifest.json:35]: Please also include an image of your dashboard with data populated.

File: sym/README.md (hunk 8 above)
Comment (llam15) [reply sym/README.md:30]: Sym customers have access to a `symflow` CLI, which can be used to verify their Sym resources:

https://docs.symops.com/docs/manage-resources

File: sym/README.md [hunk 15]
```diff
@@ -0,0 +1,37 @@
+# Sym Log Destination
+
+## Overview
+
+[Sym][1] is a platform that allows you to define simple automations that turn just-in-time access policies into easy-to-operate workflows, executed in Slack. Define access flows in Terraform, customize and integrate with other systems in code, and then use our API or Slack App to request and approve/deny access.
```
Comment (alai97) [inline sym/README.md:5]:
```suggestion
[Sym][1] is a platform that allows you to define simple automations that turn just-in-time access policies into easy-to-operate workflows, executed in Slack. Define access flows in Terraform, customize and integrate with other systems in code, and use our API or Slack App to request and approve/deny access.
```

File: sym/README.md [hunk 16]
```diff
@@ -0,0 +1,37 @@
...
+
+This integration enables customers to send Sym audit logs directly to Datadog via a Sym Log Destination.
```
Comment (alai97) [inline sym/README.md:7]:
```suggestion
This integration enables customers to send Sym audit logs directly to Datadog using a Sym Log Destination. 
```

File: sym/README.md [hunk 17]
```diff
@@ -0,0 +1,37 @@
...
+### Installation
+
+To set up the Sym integration:
+1. From the Sym Datadog Integration tile, click on "Connect Accounts".
+2. Datadog will redirect you to Sym to begin the OAuth authorization flow. Enter your Sym Org ID here to continue to log in to Sym.
+3. After successfully authenticating, a `sym_log_destination` Terraform resource will display. Copy and paste this into your Sym Terraform Configuration.
+
+### Configuration
+
+Check out the [Sym Docs][3] for detailed instructions on how to configure your Datadog Log Destination in Terraform.
```
Comment (alai97) [inline sym/README.md:22]:
```suggestion
For more information about configuring your Datadog Log Destination in Terraform, see the [Sym documentation][3].
```

**Review by alai97 — APPROVED** [review]
Looks good, some suggestions!

File: sym/README.md [hunk 18]
```diff
@@ -0,0 +1,37 @@
...
+These logs are sent in real time for every event processed by the Sym platform, such as `request` or `approve`.
+
+## Setup
+
+### Installation
+
+To set up the Sym integration:
+1. From the Sym Datadog Integration tile, click on "Connect Accounts".
+2. Datadog will redirect you to Sym to begin the OAuth authorization flow. Enter your Sym Org ID here to continue to log in to Sym.
+3. After successfully authenticating, a `sym_log_destination` Terraform resource will display. Copy and paste this into your Sym Terraform Configuration.
```
Comment (anawishnoff) [inline sym/README.md:18]:
```suggestion
3. After successfully authorizing, a `sym_log_destination` Terraform resource will display. Copy and paste this into your Sym Terraform Configuration.
```

File: sym/README.md [hunk 19]
```diff
@@ -0,0 +1,37 @@
...
+
+### Configuration
+
+For more information about configuring your Datadog Log Destination in Terraform, see the [Sym documentation][3].
+
+### Validation
+
+After you have Terraformed your Datadog Log Destination, you can confirm its existence with:
```
Comment (anawishnoff) [inline sym/README.md:26]:
```suggestion
After you have Terraformed your Datadog Log Destination, you can confirm its existence with the following command in your `symflow` CLI:
```

File: sym/README.md [hunk 20]
```diff
@@ -0,0 +1,37 @@
...
+```
+ symflow resources list sym_log_destination
+```
+
+## Troubleshooting
```
Comment (anawishnoff) [inline sym/README.md:31]: Please also add a `## Uninstallation` section here, as mentioned in [this part of the documentation](https://docs.datadoghq.com/developers/integrations/oauth_for_integrations/#publish-the-oauth-client).

File: sym/manifest.json [hunk 21]
```diff
@@ -0,0 +1,79 @@
...
+                "image_url": "images/define_deploy.jpg"
+            },
+            {
+                "media_type": "image",
+                "caption": "Request, approve, and deny anywhere in Slack",
+                "image_url": "images/request_approve.jpg"
+            },
+            {
+                "media_type": "image",
+                "caption": "An example of the Sym Overview Dashboard",
```
Comment (anawishnoff) [inline sym/manifest.json:37]: Note that images need to have at least a 16:9 aspect ratio or else they will be cut off. You can learn more in our [docs on media guidelines here](https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#media-carousel).

File: sym/manifest.json [hunk 22]
```diff
@@ -0,0 +1,79 @@
...
+                "image_url": "images/sym_overview_dashboard.png"
+            }
+        ],
+        "classifier_tags": [
+            "Supported OS::Windows",
+            "Supported OS::macOS",
+            "Supported OS::Linux",
+            "Category::Security",
```
Comment (anawishnoff) [inline sym/manifest.json:45]:
```suggestion
            "Category::Security",
            "Category::Automation",
            "Category::Developer Tools",
```

Comment (anawishnoff) [reply sym/manifest.json:45]: Suggesting a couple more categories, feel free to ignore if you disagree

**Review by anawishnoff — CHANGES_REQUESTED** [review]
A few more comments from my end. Thanks!

File: sym/README.md [hunk 23]
```diff
@@ -0,0 +1,43 @@
...
+### Validation
+
+After you have Terraformed your Datadog Log Destination, you can confirm its existence with the following `symflow` CLI command:
+```
+ symflow resources list sym_log_destination
+```
+
+### Uninstallation
+
+- Uninstall the integration by disabling the Sym application in the OAuth Apps section of your Datadog Organization settings.
```
Comment (anawishnoff) [inline sym/README.md:33]: The recommended flow to uninstall is for users to click the Uninstall button on the integration tile

**Review by anawishnoff — APPROVED** [review]
Looks great other than one minor comment!

**Review by steveny91 — APPROVED** [review]

File: sym/README.md [hunk 24]
```diff
@@ -0,0 +1,43 @@
...
+
+### Validation
+
+After you have Terraformed your Datadog Log Destination, you can confirm its existence with the following `symflow` CLI command:
+```
+ symflow resources list sym_log_destination
+```
+
+### Uninstallation
+
```
Comment (moreauowen) [inline sym/README.md:32]:
```suggestion
## Uninstallation

```

**Review by moreauowen — CHANGES_REQUESTED** [review]
Hi there! Left one comment about moving the `Uninstallation` section of the README to an H2 header. Thank you!

**Review by moreauowen — APPROVED** [review]

**Pull request merged**